test_libs/pyspec/eth2spec/test/genesis/test_validity.py

```diff
@@ -51,13 +51,14 @@ def test_is_valid_genesis_state_true_more_balance(spec):
     yield from run_is_valid_genesis_state(spec, state, valid=True)
 
 
-@with_phases(['phase0'])
-@spectest_with_bls_switch
-def test_is_valid_genesis_state_false_not_enough_balance(spec):
-    state = create_valid_beacon_state(spec)
-    state.validators[0].effective_balance = spec.MAX_EFFECTIVE_BALANCE - 1
-
-    yield from run_is_valid_genesis_state(spec, state, valid=False)
+# TODO: not part of the genesis function yet. Erroneously merged.
+# @with_phases(['phase0'])
+# @spectest_with_bls_switch
+# def test_is_valid_genesis_state_false_not_enough_balance(spec):
+#     state = create_valid_beacon_state(spec)
+#     state.validators[0].effective_balance = spec.MAX_EFFECTIVE_BALANCE - 1
```

Comment:
Does this test pass?

Reply:
No, see https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#genesis-state
No balance condition is present. So there's no such thing as "not enough balance"